src/courtroom.cpp

```diff
@@ -2260,18 +2250,12 @@ bool Courtroom::handle_objection()
             "custom_objections/" + custom_objection.split('.')[0],
             m_chatmessage[CHAR_NAME],
             ao_app->get_char_shouts(m_chatmessage[CHAR_NAME]));
-        shout_message = ao_app->read_char_ini(f_char, custom_objection.split('.')[0] + "_message", "Shouts");
-        if (shout_message == "")
-          shout_message = custom_objection.split('.')[0];
       }
       else {
         filename = "custom";
         objection_player->play(
             "custom", m_chatmessage[CHAR_NAME],
             ao_app->get_char_shouts(m_chatmessage[CHAR_NAME]));
-        shout_message = ao_app->read_char_ini(f_char, "custom_message", "Shouts");
-        if (shout_message == "")
-          shout_message = tr("CUSTOM OBJECTION!");
```

Comment:
This looks to be the result of a bad merge. This should have functionality attached to it - see #356

Reply:
What confuses me is that the functionality does exist--in another function :thinking:  
`log_chatmessage`: https://github.com/AttorneyOnline/AO2-Client/blob/af25625c047fc0cd1ba9a87219503ed532ee1d28/src/courtroom.cpp#L2054

Reply:
No wait, I see the issue now, it should be appending here instead of logging

Reply:
I take that back, it exists in `display_log_chatmessage`